lua/lean/infoview.lua

```diff
@@ -184,9 +184,11 @@ function Infoview:is_empty()
 end
 
 --- Close all open infoviews (across all tabs).
-function infoview.close_all()
+function infoview.close_all(pre_close_hook, post_close_hook)
```

Comment:
Is there really a use case for this? It seems to me to complicate a rather simple function.